### PR TITLE
Updated find an issue with new labels

### LINKED ISSUE #16156: Update "find an issue" page with new labels
The new labels are now present, as decided on in https://github.com/docker/docker/pull/14719, and described in [project/ISSUE-TRIAGE.md](https://github.com/docker/docker/blob/master/project/ISSUE-TRIAGE.md), so the ["find an issue"](https://github.com/docker/docker/blob/master/docs/project/find-an-issue.md) page should be updated to reflect the new labels.

### PR DESCRIPTION
**Pull request by lyndaoleary**
Fixes #16156
Signed-off-by: Lynda O'Leary lyndaoleary29@gmail.com

### CODE REVIEW

File: docs/project/find-an-issue.md [hunk 1]
```diff
@@ -42,68 +42,94 @@ weight=2
...
+<td class="tg-031e">Kind Label</td>
+<td class="tg-031e">Description</td>
+</tr>
+</thead>
+  <tbody>
     <tr>
-      <td class="tg-031e">Label</td>
-      <td class="tg-031e">Experience level guideline</td>
+      <td class="tg-031e"><strong class="gh-label kindbug">kind/bug</strong></td>
+      <td class="tg-031e">These issues contain bugs. The cause may or may not be known at triage time so debugging should be taken account into the time estimate.</td>
```
Comment (thaJeztah) [inline docs/project/find-an-issue.md]: "debugging should be taken account into..." I think that should be "taken into account.."?

File: docs/project/find-an-issue.md [hunk 2]
```diff
@@ -42,68 +42,94 @@ weight=2
...
 # Find and claim an issue
 
-On this page, you choose the issue you want to work on. As a contributor, you can work
-on whatever you want. If you are new to contributing, you should start by
-working with our known issues.
+As a contributor, you can work on any open issue you want. You can view issues in the Issues tab in every repository.
+If you are new to contributing, use the filter option to find suitable issues. You can filter issues by Author, Labels, Milestones, and Assignee or sort by time such as newest and oldest. 
 
 ## Understand the issue types
+To help you identify the different types of issues, our maintainers assign labels to issues. Labels are color-coded and help you categorize and filter issues. There are four labels categories, kind, area, experience, and priority. You can filter using one or more labels. The kind and experience labels are useful for new contributors.
```
Comment (thaJeztah) [inline docs/project/find-an-issue.md]: should the comma be a semi-colon after "categories"?

also, could you wrap this paragraph to 80 chars?

File: docs/project/find-an-issue.md [hunk 3]
```diff
@@ -42,68 +42,94 @@ weight=2
...
-      <td class="tg-031e">You have made less than ten contributions in your life time to any open source project.</td>
+      <td class="tg-031e"><strong class="gh-label kinddocs">kind/docs</strong></td>
+      <td class="tg-031e">These issues contain documentation, man pages, articles, blogs, or other significant word-driven task.
+      </td>
     </tr>
     <tr>
-      <td class="tg-031e"><strong class="gh-label novice">exp/novice</strong></td>
-      <td class="tg-031e">You have made more than ten contributions to an open source project or at least 5 contributions to Docker.  </td>
+      <td class="tg-031e"><strong class="gh-label enhancement">kind/enhancement</strong></td>
+      <td class="tg-031e">These issues contain enhancements that are not bugs or new features but can drastically improve usability or performance of a project component.
```
Comment (thaJeztah) [inline docs/project/find-an-issue.md]: "drastically" may be a bit too much :smile: (I think)

File: docs/project/find-an-issue.md [hunk 4]
```diff
@@ -42,68 +42,94 @@ weight=2
...
     <tr>
-      <td class="tg-031e"><strong class="gh-label master">exp/master</strong></td>
-      <td class="tg-031e">You have made more than 20 commits to Docker and greater than 1000 code lines or 3000 documentation lines.</td>
+      <td class="tg-031e"><strong class="gh-label intermediate">exp/intermediate</strong></td>
+      <td class="tg-031e">You understand the core concepts of Docker and are comfortable with Golang. 
+      These issues are for intermediates who want to dive deeper into the project.</td>
+    </tr>
+    <tr>
+      <td class="tg-031e"><strong class="gh-label expert">exp/expert</strong></td>
+      <td class="tg-031e">You are proficient with Docker and Golang and are activey involved in the community. These issues are for experts who understand the rationale behind design decisions and where the project is headed.
```
Comment (thaJeztah) [inline docs/project/find-an-issue.md]: typo: "activey" -> "actively"

**Comment by thaJeztah** [comment]
Terribly sorry for the delay here, @lyndaoleary, I thought I already commented on this one.

This PR looks to need a rebase (GitHub says there are merge conflicts). I like the things you did here, but I want @moxiegirl to have a look at them as well, because it's possible some things were written the way they were for a specific reason.

Thanks so much for doing this!

**Comment by moxiegirl** [comment]
These files have moved to https://github.com/docker/opensource/tree/master/docs can you reopen your PR there please.  (Also when I attempt to make them I'm getting a sphinx error.)

**Pull request closed without merging**